app/lib/methods/getRoomInfo.ts

```diff
@@ -1,7 +1,8 @@
+import { IRoom } from '../../definitions';
 import { getSubscriptionByRoomId } from '../database/services/Subscription';
 import RocketChat from '../rocketchat';
 
-const getRoomInfo = async rid => {
+const getRoomInfo = async (rid: string): Promise<Partial<IRoom> | null> => {
```

Comment:
I don't know if use a `Partial` here is the best approach, take a look at the lint erros.

Reply:
It's a room, just that `getRoomInfo` returns less information than what the `IRoom` interface does, that's why lint complains, we defined `this.room` with an object and some properties from `IRoom`, but on the context we're using it is actually a room with less attributes, that's why I opted for `Partial<IRoom>`.

Reply:
So you can use the `Pick<Type, keys>`. 
Using the keys required at RoomInfo
E.g: Pick<IRoom, 'id' | 'name'>